test/walkthrough.test.ts

```diff
@@ -101,10 +138,7 @@ describe('entire voter flow using OTP authorization', () => {
         'voteSubmissionId'
       )
       expect(receipt.previousBoardHash.length).to.eql(64)
-
-      if(USE_MOCK)
```

Comment:
This caused the tests to break with following error

`AssertionError [ERR_ASSERTION]: Mocks not yet satisfied:
GET http://us-avx:3000/mobile-api/us/config
      + expected - actual`

Can I remove it?

Reply:
Should be fixed now.